docs/how-to-guides/process-image-in-flow.md

```diff
@@ -0,0 +1,56 @@
+# Process image in flow
```

Comment:
How about put this as a sub page inside [Develop a flow](https://microsoft.github.io/promptflow/how-to-guides/develop-a-flow/index.html)

Reply:
Seems the structure of that page is not quite suitable for putting this in?
![image](https://github.com/microsoft/promptflow/assets/10575286/a17cb14d-82e1-43f4-8e0d-a469e8c907ba)